CLAUDE.md

```
@@ -1,107 +1,93 @@
-npm run lint:css     # Stylelint
+# Testing
+npm run test:unit   # JS unit tests
+npm run test:e2e    # Playwright e2e
+npm run lint:js     # ESLint
+npm run lint:php    # PHPCS + PHPStan
 ```
 
-## Architecture Overview
+## Architecture
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)